calibrators/marker_radar_lidar_calibrator/include/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.hpp

```diff
@@ -263,9 +287,15 @@ class ExtrinsicReflectorBasedCalibrator : public rclcpp::Node
   std::vector<Track> active_tracks_;
   std::vector<Track> converged_tracks_;
 
+  // Converged points
+  pcl::PointCloud<PointType>::Ptr lidar_points_ocs_;
```

Comment:
In the following code:

```
output_metrics_.clear();
estimateTransformation();
crossValEvaluation();
publishMetrics();
```

`lidar_points_ocs_` and its radar counterpart are filled and used inside a method inside `estimateTransformation` and used in another method inside `crossValEvaluation`.

When the code block in question (create pairs using `getPointsSet`) is not that heave and not kept between "iterations" it improves readability not using method variables

Reply:
Thanks! refactor a bit in https://github.com/tier4/CalibrationTools/pull/180/commits/a62aae70e5cafec37f3d28a9270e65a17b4e33c4